src/main/java/org/jenkinsci/plugins/buildnameupdater/BuildNameUpdater.java

```diff
@@ -161,7 +162,6 @@ public DescriptorImpl getDescriptor() {
     private static class MyFileCallable implements FilePath.FileCallable<String> {
         private static final long serialVersionUID = 1L;
 
-        @Override
```

Comment:
Starting with Java 6 it's allowed and recommended to put `@Override` on implementations of interface methods, see this discussion for explanation:

http://stackoverflow.com/questions/212614/should-we-override-an-interfaces-method-implementation